cmd/kepler/main.go

```diff
@@ -22,12 +22,19 @@ import (
 	"os"
 	"os/signal"
 
+	"github.com/alecthomas/kingpin/v2"
 	"github.com/oklog/run"
+	"github.com/sustainable-computing-io/kepler/internal/config"
 	"github.com/sustainable-computing-io/kepler/internal/version"
 )
 
 func main() {
-	logger := setupLogger("info", "text")
+	// parse args and config and exit with error if there is an error
+	cfg, err := parseArgsAndConfig()
```

Comment:
depending on the log level, dump the config in logs after parsing.

Reply:
good idea!

Reply:
implemented 

```
time=2025-04-03T01:39:20.984-04:00 level=INFO source=cmd/kepler/main.go:91 msg="Kepler version information" version=bb4aeacd-dirty buildTime=2025-04-03_05:39:20 gitBranch=feat-config gitCommit=bb4aeacd goVersion=go1.22.12 goOS=linux goArch=amd64

Configuration
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
log:
    level: info
    format: text

━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
```

Reply:
If the logging is text, and the level is info, we print the config in yaml format if possible (green path) or else manually build the configuration string.